### ISSUE TITLE
may I include this in Template 10?

### ISSUE THREAD

**Issue by JerryNixon** [Team-FLUI/FluiFramework#2]
First of all, brilliant. 

Second, as I am updating the Template 10 samples, I would like to include this. May I?

**Comment by ikarago** [comment]
Thanks, and absolutely! The more accessible it gets to (new) developers the better, so we all get more Fluent looking apps. 😊

**Comment by touseefbsb** [comment]
@JerryNixon  I heard prism team officially dropped the support for uwp and I also see u bringing back template10 so how would that work?

**Comment by JerryNixon** [comment]
Cool @ikarago, would only use it in the samples.

**Comment by JerryNixon** [comment]
How would what work @touseefbsb?

**Comment by touseefbsb** [comment]
I mean would you use a fixed version of prism? because with the support being dropped you wont be able to upgrade to any new prism versions.

**Comment by JerryNixon** [comment]
No. Don't see any problems with using Prism.Core. Not yet at least. If you see something I don't see, something that makes it incompatible. Just point it out to me quick-like.

**Comment by touseefbsb** [comment]
no compatibility wont be an issue right now it will work, the only limitation is that new updates and next version of prism will not come to prism uwp version, but I guess core works with all and will keep getting updates, so that might be fine
